recipes/traits/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "4.5.0" %}
+
+package:
+  name: traits
+  version: {{ version }}
+
+source:
+  fn: traits-4.5.0.tar.gz
+  url: https://pypi.python.org/packages/source/t/traits/traits-4.5.0.tar.gz
+  md5: 3ad558eebaedc63c29c80183c0371d2f
+
+build:
+  number: 0
+  script: python setup.py install
```

Comment:
After looking into it, seems this uses `setuptools`. So could we please add these flags `--single-version-externally-managed --record record.txt`?